updater/Gemfile

```diff
@@ -35,6 +35,9 @@ gem "sentry-opentelemetry", "~> 5.16"
 gem "sentry-ruby", "~> 5.16"
 gem "terminal-table", "~> 3.0.2"
 
+gem "flamegraph", "~> 0.9.5"
+gem "stackprof", "~> 0.2.16"
```

Comment:
FYI this is already in common, since we have the functionality in the dry-run script: https://github.com/dependabot/dependabot-core/blob/1fc9a86809361f0f8a121d2b8cd31d77b80e03f4/common/dependabot-common.gemspec#L57

Reply:
Do you recommend I remove it here and switch that line in common to be `spec.add_dependency`?